src/Entity/ListBuilder/DeveloperAppListBuilderForDeveloper.php

```diff
@@ -177,16 +177,16 @@ public function render() {
   }
 
   /**
-   * Redirects users to their My apps page.
+   * Redirects users to their Apps page.
    *
    * This controller assumes that it is only invoked for authenticated users.
-   * This is enforced for the 'apigee_edge.user.my_apps' route with the
+   * This is enforced for the 'apigee_edge.user.apps' route with the
    * '_user_is_logged_in' requirement.
    *
    * @return \Symfony\Component\HttpFoundation\RedirectResponse
-   *   Returns a redirect to the My apps of the currently logged in user.
+   *   Returns a redirect to the Apps of the currently logged in user.
    */
-  public function myAppsPage(): RedirectResponse {
+  public function appsPage(): RedirectResponse {
```

Comment:
If this would not be a Drupal module, this could be a BC breaking change because we change the name of a public method.
Although, we can say it is not because of this: https://www.drupal.org/core/d8-bc-policy#controllers